extension.js

```diff
@@ -42,7 +42,6 @@ class ExtensionDBusInterface {
 
         Main.wm.skipNextEffect(current_window.get_compositor_private());
         current_window.unmaximize(Meta.MaximizeFlags.VERTICAL);
-
```

Comment:
Please remove whitespace-only changes (i. e. there should be no blocks where only an empty line is added or removed)

Reply:
Sorry, maybe I wasn't clear enough. I didn't ask to reformat the whole file. I asked to get rid of changes like this line: where you remove (or add) an empty line in the middle of the code that is otherwise unchanged.

Reply:
You were perfectly clear.

What's happening is:
* My editor uses prettier as default. So I disable formatting altogether for this project so that existing formatting doesn't break.
* I saw that you're using eslint, so before push, I run `eslint --fix` so that all the files get formatted in your way. However some lines are getting like the one you review, I don't know why. So I jut reverted them to original styles.